post-scan-actions/aws-python-promote-or-quarantine/template.yml

```
@@ -68,6 +85,11 @@ Parameters:
     Description: |
       [Optional] Apply an access control list (ACL) on the file after it has been promoted or quarantined.
       (Options: private, public-read, public-read-write, authenticated-read, aws-exec-read, bucket-owner-read, bucket-owner-full-control)
+  PermissionsBoundary:
+    Type: String
+    Default: ''
+    Description: |
+      [Optional] The ARN of the IAM permission boundary. Leave it blank to disable using a permission boundary.
```

https://github.com/trendmicro/cloudone-filestorage-deployment-templates/blob/master/aws/FSS-Scanner-Stack.template#L149
```suggestion
      [Optional] The ARN of the policy used to set the permissions boundary for all the roles that will be created. Leave it blank to disable using a permissions boundary.
```